.github/workflows/test2.yml

```diff
@@ -0,0 +1,42 @@
+name: example-workflow
+on:
+  push:
+    branches:
+      - testing-action
+jobs:
+  run-anchor-build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20"
+          cache: "npm"
+
+      - name: Install dependencies
+        run: |
+          npm i -g yarn
+          yarn install
```

Comment:
I remember it was possible to specify directly on `actions/setup-node@v4` that you want yarn as the package manager. Can you check if this is still possible? That way you don't need to install yarn manually `npm i -g yarn`, and also we get some caching benefits

Reply:
Yeah docker image have yarn installed init.👌